src/cli-client/internals/cli/cli_main.go

```diff
@@ -0,0 +1,57 @@
+package cli
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/canonical/go-flags"
+	"github.com/canonical/oci-factory/cli-client/internals/logger"
+)
+
+// Example: oci-factory upload -y --release tracks=1.0-22.04,risks=edge,eol=2025-06-01
```

Comment:
I think comments like this in the code not attached to anything can easily get outdated. If you want to include examples it may be better to do it in the CLI help which will also be more useful for the end user.